segmentation/utils/visualize_test.py

```diff
@@ -10,3 +16,26 @@ def test_visualize_z_scores():
     # Assign random phenotype titles
     pheno_titles = [chr(i) for i in range(ord('a'), ord('z') + 1)]
     plot = visualize.visualize_z_scores(z, pheno_titles)
+
+
+def test_get_sort_data():
+    rand_type = random.choices(string.ascii_lowercase, k=100)
+    ids = random.choices(range(1, 10), k=100)
+    random_data = pd.DataFrame.from_dict({"PatientID": ids, "cell_type": rand_type})
+    print("Before sorting")
+    print(random_data)
+    print("----------------------------------")
+    print("After sorting")
+    print(visualize.get_sorted_data(random_data, "PatientID", "cell_type"))
```

Comment:
Test functions shouldn't verify by using print statements. Instead, you should use assertions to make sure random_data is getting sorted correctly.